pkg/shp/flags/flags.go

```diff
@@ -66,55 +64,97 @@ const (
 	RetentionTTLAfterFailedFlag = "retention-ttl-after-failed"
 	// RetentionTTLAfterSucceededFlag command-line flag.
 	RetentionTTLAfterSucceededFlag = "retention-ttl-after-succeeded"
+	// NodeSelectorFlag command-line flag.
+	NodeSelectorFlag = "node"
```

Comment:
Recommend calling this flag `node-selector` instead. "Node" by itself is ambiguous to me (it could refer to the name of the node, for example).

```suggestion
	// NodeSelectorFlag command-line flag.
	NodeSelectorFlag = "node-selector"
```

Reply:
Sure will do! I use `--node` as per of the [specification given on SHIP-0039](https://github.com/shipwright-io/community/blob/main/ships/0039-build-scheduler-opts.md#command-line-enhancements)

Will update it to `node-selector`